runtime/src/bank.rs

```diff
@@ -2198,6 +2204,12 @@ impl Bank {
                         self.feature_set.clone(),
                     );
 
+                    transaction_logs.push(
```

Comment:
I think now is a good time to truncate long log messages now that we will be storing them in rocksdb and bigtable. Maybe 100KB? (it's 1.3KB to log a single public key right now)